src/integrated/soil_snow_model.jl

```diff
@@ -379,6 +379,36 @@ function soil_boundary_fluxes!(
         ) +
         p.excess_heat_flux +
         p.snow.snow_cover_fraction * p.ground_heat_flux
+

```

Comment:
Since this section of code is the same for LandModel, SoilCanopyModel, and SoilSnowModel, it would be good to pull it out into a shared location to remove duplicated code. Maybe it could be two functions like `get_rho_sfc` and `get_local_geometry_faceN` since accessing those two is identical between the 3 models. The `dfluxBCdY` expressions vary a little bit (with vs without snow), so we can leave those in these model-specific functions. What do you think?

Reply:
(just saw that it's also used for EnergyHydrologyModel)

Reply:
yes! it's a good idea. I'll do that